Lib/unittest/case.py

```diff
@@ -98,6 +99,10 @@ def skip_wrapper(*args, **kwargs):
         test_item.__unittest_skip__ = True
         test_item.__unittest_skip_why__ = reason
         return test_item
+    if isinstance(reason, types.FunctionType):
+        test_item = reason
+        reason = 'unconditionally'
```

Comment:
I think it would be better if set `reason` to `None`.

Reply:
I would prefer to have the reason to `""`, this will be also [similar](http://junit.sourceforge.net/javadoc/org/junit/Ignore.html) to the Java world.

Reply:
I'm fine with that.  I suggested "unconditionally" mostly just because I had no better suggestion; the empty string probably should have been what I suggested in the first place :)